hazelcast-jca/pom.xml

```diff
@@ -152,32 +152,32 @@
         <dependency>
             <groupId>javax.enterprise</groupId>
             <artifactId>cdi-api</artifactId>
-            <version>1.0-SP1</version>
+            <version>2.0</version>
         </dependency>
 
         <dependency>
             <groupId>org.jboss.arquillian.container</groupId>
             <artifactId>arquillian-glassfish-embedded-3.1</artifactId>
-            <version>1.0.0.CR4</version>
+            <version>1.0.2</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.jboss.arquillian.junit</groupId>
             <artifactId>arquillian-junit-container</artifactId>
-            <version>1.0.3.Final</version>
+            <version>1.2.0.Final</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>nl.jqno.equalsverifier</groupId>
             <artifactId>equalsverifier</artifactId>
-            <version>1.7.3</version>
+            <version>3.1.9</version>
             <scope>test</scope>
         </dependency>
 
         <dependency>
-            <groupId>org.glassfish.extras</groupId>
+            <groupId>org.glassfish.main.extras</groupId>
             <artifactId>glassfish-embedded-all</artifactId>
-            <version>3.1</version>
+            <version>5.1.0</version>
```

Comment:
What are all those dependency updates? Are we breaking any behavior or compatibility?

Reply:
Can you also check if any README updates are needed based on those changes and see if code sample works fine if we have one?

Reply:
When I tried to build I saw that the failures were originated from not Hazelcast-JCA but other external packages used in the project. It was difficult to trace the errors but I ended up with such a situation and the cause of the problem is possibly that: 

Since `glassfish-embedded-all` 3.1.2 release, the artifact has been moved to `org.glassfish.main.extras` from `org.glassfish.extras`.

Thus I decided to update all the versions used in the project and it actually worked and fixed the failure. I didn't see any broken or changed behavior.

In README, I only noticed that `glasfish v3` to `glassfish v5` modification might be necessary.

Reply:
There is a code sample [here](https://github.com/hazelcast/hazelcast-code-samples/tree/master/hazelcast-integration/jca-ra).

I only tried to deploy on GlassFish but the specified endpoint (`http://localhost:8080/hazelcast-jca-example/Hello`) responses 404. Looks like a fix for servlet mapping is required.